lib/Controller/AccountsController.php

```diff
@@ -365,8 +369,10 @@ public function create(string $accountName,
 			);
 		}
 		try {
+			$account = $this->setup->createNewAccount($accountName, $emailAddress, $imapHost, $imapPort, $imapSslMode, $imapUser, $imapPassword, $smtpHost, $smtpPort, $smtpSslMode, $smtpUser, $smtpPassword, $this->currentUserId, $authMethod);
+			$this->mailboxSync->sync($account, $this->logger);
```

Comment:
suggestion: add try-catch and carry on regardless of errors. mailbox sync should not block the creation of an account